src/UI/sd-mobile/assets/images/README.md

```
@@ -0,0 +1,23 @@
+# sportDeets — App Icon & Brand Mark
+
+Italic bold **S** knocked out of a capital **D**, centered in the bowl,
+on a near-black rounded square. Icon and wordmark are both italic.
+
+## Colors
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)